src/bornhack/environment_settings.py.dist.dev

```diff
@@ -32,6 +32,9 @@ CAMP_REDIRECT_PERCENT = 40
 MEDIA_ROOT = os.path.join(
     os.path.dirname(os.path.dirname(os.path.dirname(__file__))), "media"
 )
+STATIC_ROOT = os.path.join(
```

Comment:
I think we should stick to using `pathlib.Path` as in settings.py

Reply:
That's fine